src/css/math.less

```diff
@@ -282,6 +282,18 @@
   .mq-vector-stem {
     display: block;
   }
+  
+  .mq-hat-prefix {
+    display: block;
+    text-align: center;
+    line-height: .95em;
+    margin-bottom: -.7em;
+    transform:scale(1.5,1);
```

Comment:
the world is close but not quite ready to use only un-prefixed `transform`, especially for something like this where being backwards-compatible is harmless

http://caniuse.com/transforms2d